projects/bp-gallery/src/components/views/uploads/UploadsView.tsx

```diff
@@ -53,14 +54,16 @@ const UploadsView = () => {
             <Add />
             {t('curator.createArchive')}
           </Button>
-          <PictureScrollGrid
-            queryParams={{ collections: { id: { null: true } } }}
-            scrollPos={scrollPos}
-            scrollHeight={scrollHeight}
-            hashbase={'uploads'}
-            uploadAreaProps={uploadAreaProps}
-            bulkOperations={[moveToCollection, bulkEdit]}
-          />
+          <ShowStats>
```

Comment:
Not sure either. However, I think one could argue that it feels more natural than in BulkEdit here. Still, I'm not sure we want Stats inside Curator-only views.